### PR TITLE
created home page

### PR DESCRIPTION
**Pull request by khorh**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  padding: 48px 24px 36px 24px;
+}
+
+nav img {
+  height: 72px;
+}
+
+nav div {
+  display: flex;
+  flex-direction: row;
```
Comment (MattW-R) [inline styles.css:64]: Unneeded declaration: default is row

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+}
+
+h3 {
+  font-size: 0.625rem;
+  line-height: 1.25rem;
+}
+
+a {
+  font-size: 0.625rem;
+  font-weight: 500;
```
Comment (MattW-R) [inline styles.css:35]: I think you have only imported weights up to 400

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  color: #aa00ff;
+}
+
+a:active {
+  color: #7b1fa2;
+}
+
+.material-icons.md-20 {
+  font-size: 0.625px;
+}
```
Comment (MattW-R) [inline styles.css:49]: Not sure what this does - could not find it being used in the chrome inspector

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+.project img {
+  width: 100%;
+}
+
+.project-details {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
```
Comment (MattW-R) [inline styles.css:103]: Toggling this made no changes for me

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+.tag {
+  font-size: 0.4375rem;
+  color: #ffffff;
+  border: none;
+  border-radius: 4px;
+  padding: 4px 8px;
+  background-color: #5a5858;
+  margin: 0 4px;
+}
```
Comment (MattW-R) [inline styles.css:120]: The .tag buttons spilled off page at very narrow resolutions - Could use .tags {flex-wrap: wrap;} & .tag {margin: 4px;} OR an alternative that made the buttons smaller/into a dropdown

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+  margin: 16px;
+}
+
+.tags {
+  display: flex;
+  align-items: center;
+}
+
+.tag {
+  font-size: 0.4375rem;
```
Comment (MattW-R) [inline styles.css:113]: Font size slightly too big for 'Stationary' button at some resolutions

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+  flex-direction: column;
+  border: 2px solid #c0bfbf;
+  border-radius: 4px;
+  margin-bottom: 2%;
+  width: 49%;
+}
+
+.project img {
+  width: 100%;
+}
```
Comment (MattW-R) [inline styles.css:97]: Toggling this didn't change anything for me

File: index.html [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+  </head>
+  <body>
+    <nav class="container-width">
+      <img src="images/logo.png" alt="logo" />
+      <div>
+        <li><a href="index.html">Projects</a></li>
+        <li><a href="about.html">About</a></li>
```
Comment (MattW-R) [inline index.html:23]: html validator doesn't like <li> inside a <div> (not <ol>/<ul>) - You could do what you want without using li, you then wouldn't need CSS declaration removing the bullet points

File: styles.css (hunk 6 above)
Comment (khorh) [reply styles.css:113]: Changed to font-size: 0.375rem

File: styles.css (hunk 5 above)
Comment (khorh) [reply styles.css:120]: I have made the tags on a different line as I realised the title may be longer for other projects.

File: styles.css (hunk 7 above)
Comment (khorh) [reply styles.css:97]: Deleted!  Again one of those where I changed my mind about the responsiveness!

File: index.html (hunk 8 above)
Comment (khorh) [reply index.html:23]: Deleted the li tags and left it as \<a> tags.

File: styles.css (hunk 4 above)
Comment (khorh) [reply styles.css:103]: Deleted!

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+
+main section {
+  width: 50%;
+  background-color: #f5f5f5;
+  padding: 32px;
+}
+
+main section:nth-child(2) {
+  background-color: #ffffff;
```
Comment (MattW-R) [inline styles.css:91]: Could probably just set nth-child(1) to #F5F5F5 as default is white

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,165 @@
...
+    height: 56px;
+  }
+
+  nav div a {
+    padding-top: 16px;
+  }
+
+  .project {
+    width: 100%;
+  }
```
Comment (MattW-R) [inline styles.css:158]: Could add some additional responsiveness to About section as it stays as 2 columns in mobile resolutions - Possibly will look nicer if switches to 1 column?

Comment (khorh) [reply styles.css:157]: Updated to include responsive design.  Thanks!

**Pull request merged**